tests/expected/function-contract/modifies/havoc_pass_reordered.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// kani-flags: -Zfunction-contracts
+
+// These two are reordered in comparison to `havoc_pass` and we expect the test case to pass still
```

Comment:
This test seems identical to `havoc_pass.rs`. Is there some order that meant to be changed?